chessBoard.js

```diff
@@ -0,0 +1,7 @@
+const size = 12;
+let hashString1 = (" #").repeat(size/2);
+let hashString2 = ("# ").repeat(size/2);
+
+for (let i = 0; i < size; i++) {
+    i % 2 === 0 ? console.log(hashString1) : console.log(hashString2);
```

Comment:
the norm for JS is to have two spaces per level of indentation, can you fix the indentation for all the files to follow the norm please?